Uplift/Views/GymDetail/GymDetailWeekView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  GymDetailCalendarView.swift
+//  Uplift
+//
+//  The header calendar used to select which content to display based on day of the week
+//
+//  Created by Phillip OReggio on 10/2/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymDetailWeekView: UIView {
+
+    // MARK: - Views
+    private var weekdayCollectionView: UICollectionView!
+
+    // MARK: - Info
+    private var days: [WeekDay] = WeekDay.allCases 
+
+    /// Current Day of today
+    private var today: WeekDay {
+        get {
+            let weekdayIndex = Calendar.current.component(.weekday, from: Date())
+            return WeekDay(index: (weekdayIndex + 5) % 7)
+        }
+    }
+
+    // MARK: - Display
+    private let daysSize = CGSize(width: 24, height: 24)
+
+    // MARK: - Public
+    var selectedDay: WeekDay?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupView()
+        setupConstraints()
+        weekdayCollectionView.selectItem(at: IndexPath(row: (today.index + 5) % 7, section: 0), animated: true, scrollPosition: .centeredHorizontally)
```

Comment:
can we make a helper function for `(index + 5) % 7` since it seems like we use it in multiple places, i.e. on line 26